src/sync/file-operations.ts

```diff
@@ -88,3 +90,8 @@ export function getHighestDbVersion(directoryDir: string, dbFileName: string): n
   }
   return version;
 }
+
+async function runDbFixerScript(db: Knex): Promise<void> {
+  // need to do this because cli runs migrations in .ts and augur-app runs migrations in .js
+  await db.raw("update knex_migrations set name = substr(name,1, length(name)-2) || 'js';");
```

Comment:
I see what you're saying @ryanberckmans , this is supposed to operate on the output file, not the currently open database. I bet this change i made will cause restarting augur-node to try to re-run those migrations. We should still construct a new real Knex object instead of exec'ing